### PR TITLE
Congratulations comment for GitLab

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Place "Congratulations ..." comment also on GitLab.

### CODE REVIEW

File: packit_service/worker/handlers/fedmsg_handlers.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
 from datetime import datetime
 from typing import Type, Optional
 
+from ansible.modules.source_control.gitlab_project import GitLabProject
```
Comment (mfocko) [inline packit_service/worker/handlers/fedmsg_handlers.py:31]: Isn't it supposed to be from `ogr`, same as `GithubProject`?

Comment (lachmanfrantisek) [reply packit_service/worker/handlers/fedmsg_handlers.py:31]: Ah, thanks for catch that!

Comment (lachmanfrantisek) [reply packit_service/worker/handlers/fedmsg_handlers.py]: Fixed

**Comment by lachmanfrantisek** [comment]
@mfocko PTAL

**Review by mfocko — APPROVED** [review]
LGTM

**Pull request merged**